jsonnet/utils/sanitize-rules.libsonnet

```
@@ -159,6 +159,12 @@ local patchedRules = [
           namespace: 'openshift-monitoring',
         },
       },
+      {
+        alert: 'NodeClockNotSynchronising',
+        labels: {
+          severity: 'critical',
+        },
+      },
```

This jsonnet snippet doesn't have any effect, the ClockNotSynced alert is in the `node-exporter` group. You need something like this (with probably better formatting):
```suggestion
    ],
  },
  {
    name: 'node-exporter',
    rules: [
      {
        alert: 'NodeClockNotSynchronising',
        labels: {
          severity: 'critical',
        },
      },
```